### PR TITLE
Added tank frame where we are showin tank and tank index

### PR DESCRIPTION
**Pull request by shreya-mishra**
image

### CODE REVIEW

File: src/components/TankFrame.tsx [hunk 1]
```diff
@@ -1,8 +1,32 @@
-import {View} from 'react-native';
+import {Dimensions, StyleSheet, Text, View} from 'react-native';
 import React from 'react';
+import Tank from './Tank';
 
-const TankFrame = () => {
-  return <View>{/* <Tank tankLevel={tankLevel}/> */}</View>;
+const {width} = Dimensions.get('screen');
+
+const TankFrame = ({level}: {level: number}) => {
```
Comment (ankushdharkar) [inline src/components/TankFrame.tsx:7]: What is 'level'?

File: src/components/TankFrame.tsx [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
+
+const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    width: width / 5,
+    alignItems: 'center',
+    margin: 20,
+  },
+  basket: {
```
Comment (ankushdharkar) [inline src/components/TankFrame.tsx:26]: What is a 'basket'?

File: src/components/TankFrame.tsx (hunk 1 above)
Comment (shreya-mishra) [reply src/components/TankFrame.tsx:7]: committed in comments/resolved PR

File: src/components/TankFrame.tsx (hunk 2 above)
Comment (shreya-mishra) [reply src/components/TankFrame.tsx:26]: resolved

**Pull request merged**